e2e/cypress/support/api_commands.js

```diff
@@ -466,8 +549,6 @@ Cypress.Commands.add('apiUnpinPosts', (postId) => {
 // *****************************************************************************
 
 Cypress.Commands.add('apiUpdateConfig', (newSettings = {}) => {
-    cy.apiLogin('sysadmin');
```

Comment:
Removed a few instances of where we were logging in and out as the sysadmin. Depending on the test, these logins were extraneous as the test was already logged in. Logging out then could also be problematic if you were not aware that it would happen. I think the tests themselves should be responsible for doing logging in and out actions, so it's clear what users are doing specific things.

Reply:
I'm fine with it but can it be on a separate PR?

Reply:
For sure, I'll make it part of the `mm` module refactor as that aligns pretty well with that since I'm mucking with all the API call code.

Reply:
All instances of this refactoring should have been removed from this PR.